docs/getting-started.txt

```diff
@@ -27,75 +28,117 @@ code.
 Software
 ========
 
--  Determined agent and master nodes must be configured with Ubuntu 16.04 or higher, CentOS 7, or
-   macOS 10.13 or higher.
+-  Determined *agent* and *master* nodes must be configured with Ubuntu 16.04 or higher, CentOS 7,
+   or macOS 10.13 or higher.
 
 -  Agent nodes must have Docker installed.
 
--  To run jobs with GPUs, Nvidia drivers, version 384.81 or higher, must be installed on each agent.
-   Nvidia drivers can be installed as part of a CUDA installation but the rest of the CUDA toolkit
-   is not required.
+-  To run jobs with GPUs, install Nvidia drivers, version 384.81 or higher, on each agent. The
+   drivers can be installed as part of a CUDA installation but the rest of the CUDA toolkit is not
+   required.
 
 Hardware
 ========
 
--  At least 4-CPU cores are recommended for the master node, Intel Broadwell or later, and 8GB RAM,
-   and 200GB of free disk space. The master node does not require GPUs.
+-  Master node:
+
+   -  At least 4-CPU cores, Intel Broadwell or later. The master node does not require GPUs.
+   -  8GB RAM
+   -  200GB of free disk space.
+
+-  Agent Node:
 
--  at least 2-CPU cores are recommended for each agent node, Intel Broadwell or later, and 4GB RAM,
-   and 50GB of free disk space. If you are using GPUs, Nvidia GPUs with compute capability 3.7 or
-   greater are required, such as K80, P100, V100, A100, GTX 1080, GTX 1080 Ti, TITAN, and TITAN XP.
+   -  At least 2-CPU cores, Intel Broadwell or later.
+   -  If you are using GPUs, Nvidia GPUs with compute capability 3.7 or greater are required: K80,
+      P100, V100, A100, GTX 1080, GTX 1080 Ti, TITAN, or TITAN XP.
+   -  4GB RAM
+   -  50GB of free disk space.
 
 Docker
 ======
 
-For the examples that use an agent, install Docker to run containerized workloads. If you do not
-already have Docker installed, follow the :doc:`Installing Docker
+For the distributed example that uses an agent, install Docker to run containerized workloads. If
```

Comment:
Logic error, I guess. So, we don't even need to say "that use agent" because they all do. From the wording of the examples, it seemed like only example 2 used an agent.

Reply:
Done.